src/tests/test_pyproxy.py

```diff
@@ -646,68 +684,79 @@ def test_pyproxy_mixins32(selenium, configurable, writable):
     )
 
 
+@run_in_pyodide
 def test_pyproxy_mixins41(selenium):
-    selenium.run_js(
-        """
-        [Test, t] = pyodide.runPython(`
-            class Test:
-                caller="fifty"
-                prototype="prototype"
-                name="me"
-                length=7
-                def __call__(self, x):
-                    return x + 1
+    from pyodide.code import run_js
+    from pyodide.ffi import to_js
 
-            from pyodide.ffi import to_js
-            to_js([Test, Test()])
-        `);
-        assert(() => Test.$prototype === "prototype");
-        assert(() => Test.prototype === "prototype");
-        assert(() => Test.name==="me");
-        assert(() => Test.length === 7);
-
-        assert(() => t.caller === "fifty");
-        assert(() => "prototype" in t);
-        assert(() => t.prototype === "prototype");
-        assert(() => t.name==="me");
-        assert(() => t.length === 7);
-        assert(() => t(7) === 8);
-        Test.destroy();
-        t.destroy();
+    class Test:
+        caller = "fifty"
+        prototype = "prototype"
+        name = "me"
+        length = 7
+
+        def __call__(self, x):
+            return x + 1
+
+    test_objects = to_js([Test, Test()])
+
+    run_js(
         """
-    )
+        ([Test, t]) => {
+            assert(() => Test.$prototype === "prototype");
+            assert(() => Test.prototype === "prototype");
+            assert(() => Test.name==="me");
+            assert(() => Test.length === 7);
+
+            assert(() => t.caller === "fifty");
+            assert(() => "prototype" in t);
+            assert(() => t.prototype === "prototype");
+            assert(() => t.name==="me");
+            assert(() => t.length === 7);
+            assert(() => t(7) === 8);
+            Test.destroy();
+            t.destroy();
+        }
+        """
+    )(test_objects)
 
 
+@run_in_pyodide
 def test_pyproxy_mixins42(selenium):
-    selenium.run_js(
-        """
-        let t = pyodide.runPython(`
-            class Test:
-                def __call__(self, x):
-                    return x + 1
+    from pyodide.code import run_js
+    from pyodide.ffi import to_js
 
-            from pyodide.ffi import to_js
-            Test()
-        `);
-        assert(() => "prototype" in t);
-        assert(() => t.prototype === undefined);
-        t.destroy();
+    class Test:
+        def __call__(self, x):
+            return x + 1
+
+    t = Test()
+
+    run_js(
         """
-    )
+        (t) => {
+            assert(() => "prototype" in t);
+            assert(() => t.prototype === undefined);
+            t.destroy();
+        }
+        """
+    )(to_js(t))
```

Comment:
```suggestion
    )(t)
```

Not using `to_js` here is the original intention I think.

Reply:
The `to_js` call doesn't do anything so the overall behavior is the same, but it is confusing.

Reply:
Ack